variables.tf

```diff
@@ -95,7 +100,7 @@ variable "postgres_public_access_cidrs" {
 }
 
 variable "ssh_public_key" {
-  default = "~/.ssh/id_rsa.pub"
+  default = null
```

Comment:
Minor change, need to update the "default" column for this variable in the CONFIG-VARS.md

Reply:
done